src/util/JoinAlgorithms/FindUndefRanges.h

```diff
@@ -174,7 +184,7 @@ struct FindSmallerUndefRanges {
   CPP_template(typename Row,
                typename It)(requires std::random_access_iterator<It>) auto
   operator()(const Row& row, It begin, It end,
-             bool& resultMightBeUnsorted) const -> cppcoro::generator<It> {
+             bool& resultMightBeUnsorted) const {
```

Comment:
This functor does not compile as the return type is different. One option would be to remove this functor, alternatevely, any idea how to make the return type of the 3 methods equal?